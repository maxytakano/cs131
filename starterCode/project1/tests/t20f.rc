structdef _MYS_1 { 
    int x;
    float y; 
    _MYS_1 *m1;
    function : float f( int &x ) {}
    function : void foo() {} 
};

structdef _MYS_2 { 
    int x;
    bool y; 
    _MYS_2 *m2;
    function : void f( int &x ) {} 
    function : void foo() {} 
};

function : int main()
{
  _MYS_1 x;
  _MYS_2 y;
  return (int) (y.y);
}
