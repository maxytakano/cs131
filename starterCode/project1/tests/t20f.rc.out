Error, "/tmp/cs131_testrunner_sJTaux/file.rc", line 5: 
  Return required in function, none found.
Compile: failure.
